docs/advertising-details.rst

```diff
@@ -110,13 +110,50 @@ However, we always give advance notice in our issue tracker
 and via email about showing ads where none were shown before.
 
 
+.. _do-not-track:
+
+Do Not Track Policy
```

Comment:
Not sure this is the right part of the docs for this. It feels like it should probably be it's own thing, since it applies to RTD itself, and not just ads. 

I realize there are ad-specific things, so maybe a DNT page in the docs, and then also a section here?

Reply:
Perhaps having it in the Privacy Policy is enough as an additional section. I guess it depends how heavily we want to promote the fact that we support it.

Reply:
I think a section in the privacy policy is good and the advertising details will link there.

Reply:
👍